src/component/Empdetails.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from "react";
+import { Box, Button, Stack, Typography } from "@mui/material";
+import { collection, deleteDoc, doc, getDocs } from "firebase/firestore";
+import { auth, database } from "../firebase";
+import { NavLink, useNavigate } from "react-router-dom";
+import Image from "../assets/homeimg.jpg";
+import Navbar from '../component/Navbar'
+
+function Empdetails() {
+
+    const [value, setValue] = useState([]);
+    const navigate = useNavigate();
+  
+    useEffect(() => {
+      const getData = async () => {
```

Comment:
create a separate file for all the function which will return the response
create a util file for this